### PR TITLE
Open Source Workshop page

### PR DESCRIPTION
**Pull request by afeld**
Finally putting together the homepage for the Open Source workshop, as described here:

https://docs.google.com/document/d/1xyYIkeZ_htZ76tmrA06UakSvkOX7ghWL4RG7Gr6oIbE/edit

Note that the RSVP mechanics very closely resemble the "Bonus Round" on http://18f.github.io/hourofcode/ that @leahbannon and I put together.
## Open questions
- Does the description make sense – and do the instructions work – for someone without any context?
- What information am I forgetting?
- Do we need to know people's skill levels and/or preferred programming languages in advance?
- Do people need to RSVP through Meetup as well?
  - Feels a bit redundant, but would be nice to have a way to email attendees before/after the event. I suppose we could just leave comments on all the PRs if we need to...
- What's a better name for the event?
## TODOs
- [ ] Create event on Meetup (even if RSVPs aren't done there)
- [ ] Announce to Hacker Hours mailing list

---

/cc @gabekneisley @melodykramer @wslack @ultrasaurus @jdaudier

### CODE REVIEW

**Comment by afeld** [comment]
open source workshop hacker hours

File: oss-workshop.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Open Source Workshop
+---
+
+4/26/15 12-5pm, [Projective Space LES](http://www.projective.co/), 72 Allen St. 3rd Floor, NYC
```
Comment (afeld) [inline oss-workshop.md:5]: I put 12-5 somewhat arbitrarily – thoughts on a good amount of time, assuming we will have some orientation and maybe short presentations of what people worked on at the end?

**Comment by gabekneisley** [comment]
I think the instructions are fantastic. Clear and unambiguous, assuming you have an understanding of what github is (which I think is an appropriate baseline for this event).

The only thing missing, IMO, is that the event description presupposes that you know what open source is, want to be part of it, and can help. I don't think we want to take those things for granted. I don't know how to explain this succinctly yet, but if others agree that we should explain this, I'll be happy to write a draft. Something like:

## Open Source Workshop

Did you know you can make meaningful contributions to the code powering thousands of websites, and running on millions of browsers? Absolutely! You can improve your coding, technical writing and collaboration skills, expand your professional network, and help advance the art of software engineering all in one fun way—contribute to open source! It looks good on your resume, teaches you how to be thorough and work within teams, and gives you inside access to some of the coolest projects being built. _etc_

**Comment by jdaudier** [comment]
FYI @afeld - another event with similar cause: http://www.meetup.com/Developers-for-Good/events/221704410

Past one: http://www.writespeakcode.com/conference/code/

**Comment by jdaudier** [comment]
might be interesting to ask attendees to create an issue with what they would like to learn in this workshop

**Comment by jdaudier** [comment]
I thought this OS project might be good one to practice against: https://github.com/illacceptanything/illacceptanything

Not as final contribution since it's kind of a joke. But might be fun to use for demo purposes.

**Comment by afeld** [comment]
> the event description presupposes that you know what open source is, want to be part of it, and can help

Mmmm, good observation.

> Something like

Perfect – added!

> another event with similar cause: http://www.meetup.com/Developers-for-Good/events/221704410

Haha @courte @dbness we should talk.

> Past one

Yeah, this is definitely not a new idea. Have a bunch of examples linked at the top of [the Google Doc](https://docs.google.com/document/d/1xyYIkeZ_htZ76tmrA06UakSvkOX7ghWL4RG7Gr6oIbE/edit).

> might be interesting to ask attendees to create an issue with what they would like to learn in this workshop

Good call – added to the page!

> I thought this OS project might be good one to practice against

Good idea! Added to the Google Doc.

File: oss-workshop.md (hunk 1 above)
Comment (afeld) [inline oss-workshop.md:5]: Can I get a :+1:/:-1:?

Comment (afeld) [inline oss-workshop.md:5]: Space is available 11-6 I think...correct @gabekneisley?

**Comment by afeld** [comment]
Going to quietly merge, and will start working on the email. Still open to iterate on the page, obvs. #metaopensource

**Pull request merged**